docs/Learning-Environment-Design-Agents.md

```diff
@@ -653,26 +639,26 @@ nothing"_ or _"change weapon"_ for his next decision (since action index 1 and 2
 are masked)
 
 ```csharp
-SetMask(0, new int[2]{1,2})
+WriteMask(0, new int[2]{1,2})
 ```
 
 Notes:
 
-- You can call `SetMask` multiple times if you want to put masks on multiple
+- You can call `WriteMask` multiple times if you want to put masks on multiple
   branches.
 - You cannot mask all the actions of a branch.
 - You cannot mask actions in continuous control.
 
 ### Actions Summary & Best Practices
 
-- Actions can either use `Discrete` or `Continuous` spaces.
-- When using `Discrete` it is possible to assign multiple action branches, and
-  to mask certain actions.
+- Agents can either use `Discrete` or `Continuous` actions.
```

Comment:
```suggestion
- Agents can use `Discrete` and/or `Continuous` actions.
```

Reply:
Might punt on that for now. While technically true, it's only possible with actuators (which we don't have a section on)